test/api/v3/integration/GET-dataexport.test.js

```diff
@@ -0,0 +1,424 @@
+import moment from 'moment';
+
+import { generateUser, requester } from '../../../helpers/api-integration/v3';
+import { Task } from '../../../../website/server/models/task';
+
+describe('GET /export/userdata.xml', () => {
```

Comment:
The PR is looking good @benkelaar , I'm a bit concerned about this test because it'll require updates every time something changes in the default user schema (a new gear, a new audio theme available, a new preference...)

Reply:
ah, also we have an existing integration test for this route, https://github.com/HabitRPG/habitica/blob/develop/test/api/v3/integration/dataexport/GET-export_userdata.xml.test.js, I'm going to merge the PR removing this test

Reply:
Yeah, I was going back and forth about it as well.  Apologies that I missed that test.  Thanks for removing it.  Might take some time next week to document the test conventions somewhere.

Reply:
> Might take some time next week to document the test conventions somewhere.

That would be awesome if you want to do that! 

It could go in the tests section of [Using Your Local Install to Modify Habitica's Website and API](https://habitica.fandom.com/wiki/Using_Your_Local_Install_to_Modify_Habitica's_Website_and_API) although we may end up finding that a separate wiki page just for running and writing tests would be helpful.

Reply:
Yeah, was thinking it might be worthwhile to have a separate dedicated page for that as well.  However, our baby thought my laptop might make a good water receptacle yesterday, so had a bit of a setback work-wise 😆 .   I have a (temporary) replacement laptop now, but will have to re-do some of the things, so might take me a while.